packages/expo-symbols/src/Symbol.tsx

```diff
@@ -5,7 +5,7 @@ import { NativeSymbolViewProps, SymbolViewProps } from './SymbolModule.types';
 const NativeView: React.ComponentType<NativeSymbolViewProps> =
   requireNativeViewManager('SymbolModule');
 
-export default function SymbolView(props: SymbolViewProps) {
+export function Symbol(props: SymbolViewProps) {
```

Comment:
To be honest I liked `SymbolView` more. `Symbol` is already a built-in class in JS